rasa_sdk/utils.py

```diff
@@ -76,3 +83,82 @@ def arguments_of(func) -> List[Text]:
     """Return the parameters of the function `func` as a list of their names."""
 
     return inspect.signature(func).parameters.keys()
+
+
+def number_of_sanic_workers() -> int:
+    """Get the number of Sanic workers to use in `app.run()`.
+    If the environment variable `constants.ENV_SANIC_WORKERS` is set and is not equal to 1.
+    """
+
+    def _log_and_get_default_number_of_workers():
+        logger.debug(
+            f"Using the default number of Sanic workers ({DEFAULT_SANIC_WORKERS})."
+        )
+        return DEFAULT_SANIC_WORKERS
+
+    try:
+        env_value = int(os.environ.get(ENV_SANIC_WORKERS, DEFAULT_SANIC_WORKERS))
+    except ValueError:
+        logger.error(
+            f"Cannot convert environment variable `{ENV_SANIC_WORKERS}` "
+            f"to int ('{os.environ[ENV_SANIC_WORKERS]}')."
+        )
+        return _log_and_get_default_number_of_workers()
+
+    if env_value == DEFAULT_SANIC_WORKERS:
+        return _log_and_get_default_number_of_workers()
+
+    if env_value < 1:
+        logger.warning(
```

Comment:
Let's use `warnings` because it's something the user can and should fix (https://stackoverflow.com/a/14762106/3429596)

```suggestion
        warnings.warn(
```